app/components/cases/new-subcase.js

```diff
@@ -158,6 +158,8 @@ export default class CasesNewSubcase extends Component {
     } else {
       subcase.linkedPieces = pieces;
     }
+    // TODO KAS-3484 is there a situation where we don't have to inherit the governmentAreas?
```

Comment:
I don't think there is. Currently `governmentAreas` are changed on the case level, so they are implicitly inherited between subcases already. So this looks fine to me, as long as they can be changed per subcase by the user later on.